src/components/navigation-header/index.tsx

```diff
@@ -43,10 +46,10 @@ const NavigationHeader: React.FC = () => {
           </Link>
           <ProductSwitcher />
         </div>
-        {currentProduct?.navigationHeaderItems && (
+        {hasNavigationItems && (
           <div className={s.headerRight}>
             <ul className={s.navLinks}>
-              {currentProduct?.navigationHeaderItems.map((navLink) => {
+              {navItems.map((navLink: NavigationHeaderItem) => {
```

Comment:
The introduction of the `navItems` and `hasNavigationItems` variables helped make this code a little more readable.